### DISCUSSION TITLE
Animation help

### DISCUSSION THREAD

**Discussion by quick007** [pmndrs/react-spring#1871] [Support]
Hi all,

I'm currently trying to make an animation like this:
![image](https://user-images.githubusercontent.com/69743171/163904212-4ae5777a-1063-4550-af06-3a3814ea501b.png)

I'm not really sure the best way to make that smooth curve, and I think it would look really cool if I was able to get it that way. if I need to use janky positioning that's fine, I can just hide the animation element and put another element using normal positioning on top of it.

**Comment by J4v4Scr1pt** [comment]
Hi!

I'm to trying to learn about this awesome lib also. I stumbled across this great tutorial video that sort of does what you ask for. Guess you can play a bit with it to get it even smoother. Hope it helps :).

https://www.youtube.com/watch?v=kT6yYSwK1oA
